storage_examples_test.go

```diff
@@ -572,6 +572,183 @@ func ExampleStorage_Select_from_disk_out_of_order() {
 	// timestamp: 1600000003, value: 0.1
 }
 
+// Simulates inserting an outdated row that forces inserting into a non-head partition.
+func ExampleStorage_InsertRows_outdated() {
+	tmpDir, err := ioutil.TempDir("", "tstorage-example")
+	if err != nil {
+		panic(err)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	storage, err := tstorage.NewStorage(
+		tstorage.WithDataPath(tmpDir),
+		tstorage.WithTimestampPrecision(tstorage.Seconds),
+		tstorage.WithPartitionDuration(3*time.Second),
+	)
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err := storage.Close(); err != nil {
+			panic(err)
+		}
+	}()
```

Comment:
This is unneeded because the first one will get closed at L623

Reply:
good catch